candi/bashible/bashbooster/50_deckhouse_registrypackages.sh

```diff
@@ -12,8 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# shellcheck disable=SC2211,SC2153
+
 bb-var BB_RP_INSTALLED_PACKAGES_STORE "/var/cache/registrypackages"
-# shellcheck disable=SC2153
+bb-var BB_RP_FETCHED_PACKAGES_STORE "${TMPDIR}/registrypackages"
+
+# Use d8-curl if installed, fallback to system package if not
+bb-rp-curl() {
+  if command -v d8-curl > /dev/null ; then
+    d8-curl --parallel "$@"
+  else
+    curl "$@"
+  fi
```

Comment:
Please add one more check if default curl supports parallel downloads

Reply:
It's not necessary. `curl` is used only once to download the `d8-curl` and `jq` packages. It doesn't make much sense to download two lightweight packages in parallel.